apps/docs/functions/src/prompts/templates/recommendations.ts

```
@@ -0,0 +1,92 @@
+
+export const RECOMMENDATIONS_PROMPT: PromptTemplate = {
+  metadata: {
+    id: "reading-recommendations",
+    name: "Reading Recommendations",
+    category: "recommendation",
+    version: "1.1.0",
+    description:
+      "Suggests next articles based on reading history with semantic search",
+    createdAt: "2025-11-27",
+    author: "Phoenix AI Team",
+    changelog: "Added semantic search for contextually relevant recommendations",
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)